sql-statements/sql-statement-backup.md

```diff
@@ -18,25 +18,24 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-backup/']
 
 ## 语法图
 
-**BackupStmt:**
-
-![BackupStmt](/media/sqlgram/BackupStmt.png)
-
-**BRIETables:**
-
-![BRIETables](/media/sqlgram/BRIETables.png)
-
-**BackupOption:**
-
-![BackupOption](/media/sqlgram/BackupOption.png)
-
-**Boolean:**
-
-![Boolean](/media/sqlgram/Boolean.png)
-
-**BackupTSO:**
-
-![BackupTSO](/media/sqlgram/BackupTSO.png)
+```ebnf+diagram
+BackupStmt ::=
+    "BACKUP" BRIETables "TO" stringLit BackupOption*
```

Comment:
Will the blank lines affect the display?

Reply:
No the treats new lines as the same as spaces which are ignored. Even

``````markdown
```ebnf+diagram
a ::= b c ::= d
```
``````

will work